packages/cli/test/fixtures/unit/vercel-env-pull-delta-quotes/.env

```diff
@@ -0,0 +1,2 @@
+NEW_VAR=testvalue
+SPECIAL_FLAG=1
```

Comment:
Should this have a newline? I cant remember if we generate one or not

Reply:
Added newline 👍